src/SFA.DAS.Admin.Roatp.Web/Models/ProviderStatusUpdateViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using SFA.DAS.Admin.Roatp.Domain.Models;
+
+namespace SFA.DAS.Admin.Roatp.Web.Models;
+
+
+public class OrganisationStatusUpdateViewModel : IBackLink
+{
+    public required OrganisationStatus OrganisationStatus { get; set; }
+    public required string Ukprn { get; set; }
```

Comment:
Not sure why we need ukprn in the model, it is part of the route?

Reply:
we dont. Removed